plugins/modules/dcnm_vrf.py

```diff
@@ -429,10 +435,11 @@ def diff_for_attach_deploy(self, want_a, have_a):
                         else:
                             found = True
 
-                            if bool(have['isAttached']) is not bool(want['isAttached']):
-                                del want['isAttached']
-                                attach_list.append(want)
-                                continue
+                            if want.get('isAttached') is not None:
```

Comment:
Please add a comment here describing why `want['isAttached']` is being deleted in this case

Reply:
Added